pkg/network/payload/version.go

```diff
@@ -26,60 +20,50 @@ type Version struct {
 	Magic config.NetMode
 	// currently the version of the protocol is 0
 	Version uint32
-	// currently 1
-	Services uint64
 	// timestamp
 	Timestamp uint32
-	// port this server is listening on
-	Port uint16
 	// it's used to distinguish the node from public IP
 	Nonce uint32
 	// client id
 	UserAgent []byte
-	// Height of the block chain
-	StartHeight uint32
 	// Whether to receive and forward
 	Relay bool
```

Comment:
And BTW, this one is substituted by full node capability if I'm not missing something.

Reply:
That's true, so replaced `Relay` with full node check. Do we still need `Relay` in server configuration?

Reply:
Now we keep `Rely` in server configuration and adding `FullNode` capability depends on this flag.